rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy

```diff
@@ -3286,6 +3283,8 @@ class MenuController extends ControllerBase implements ApplicationContextAware{
                 if (frameworkService.isClusterModeEnabled()) {
                     //initialize if in another node
                     scmService.initProject(params.project, 'import')
+                    scmService.fixImportStatus(authContext, params.project, result.nextScheduled)
+                    scmService.importPluginStatus(authContext, params.project)
```

Comment:
the result of this method is never used, is this a redundant call?

Reply:
The response is not needed just to check the status for the future method to check individual file, this must be refactored on next enhancement to SCMon cluster too.

Reply:
can you file an issue about any refactoring that should happen?